setup.py

```diff
@@ -1,16 +1,38 @@
 #!/usr/bin/env python
+import io
```

Comment:
Where does it come from? I don't think I saw it before, and in https://github.com/bluesky/bluesky/pull/1331...

Reply:
Standard library https://docs.python.org/3/library/io.html

Its implementation of `io.open` is in distinct from `builtins.open` (a.k.a. `open`) in Python 2 and supports the `encoding` argument that we use here. I hope to be forgetting this archaic Python 2 knowledge soon....

Reply:
Since you check `sys.version` and exit if it's below 3.6, I don't think using `io.open` is necessary? 

Even if you use the builtin `open`, the interpreter will never get that far, and the friendly error message will be displayed. The file as a whole doesn't need to be python2 compatible; just the part that gets executed when running it with python2.

Reply:
It works fine as it is though, so this could safely be merged.

Reply:
This is right. Fixed and force-pushed.